mesonbuild/backend/vs2010backend.py

```diff
@@ -1025,8 +1025,6 @@ def gen_vcxproj(self, target, ofname, guid):
         if len(target_args) > 0:
             target_args.append('%(AdditionalOptions)')
             ET.SubElement(clconf, "AdditionalOptions").text = ' '.join(target_args)
-
-        target_inc_dirs.append('%(AdditionalIncludeDirectories)')
```

Comment:
I am not sure I understand why this has been removed.

Reply:
Split cleanup into own commit.